orion/pkg/repos/repos_util.go

```diff
@@ -27,3 +28,16 @@ func handleSqlExecResult(result sql.Result, expected int64, errorMessage string)
 	}
 	return nil
 }
+
+func appendError(errorString string, stringId string, rowId uint, err error) string {
```

Comment:
I think this function would be easier to use if the signature just had a generic string `id`, so 3 params instead of 4. In the case we need to pass a uint, just turn the id into a string. We're simply appending error strings so it's fine to cast to string in this case.

```
appendError(errorString, fmt.Sprint(rowId), err)
```